config/locales/ar.yml

```diff
@@ -19,7 +19,10 @@ ar:
   date_field: "%{gregorian_dates} / %{hijri_dates}"
   date_field_gregorian: "%{dates} (ميلادي)"
   date_field_hijri: "%{dates}(هجري)"
+  date_field_label_gregorian: ميلادي
+  date_field_label_hijri: هجري
   date_range: "%{min} - %{max}"
+  date_range_calendar: Calendar
```

Comment:
This has been asked for in the original ticket and is being tracked here: https://github.com/orgs/sul-dlss/projects/35#column-12461630

I'm hoping tracking there should enable us to move forward on review of this.